File: Ulbra 2023-1/Introdução ao Desenvolvimento Web/Aula 17 (JavaScript)/exerc4.js
//4 - Faça um script que receba um número, calcule e mostre a tabuada desse número.

let numero = parseInt(prompt("Digite um número inteiro: "));

console.log(`Tabuada de ${numero}:`);
for (let i = 1; i <= 10; i++) {
    let resultado = numero * i;
    alert(`${numero} x ${i} = ${resultado}`);
}

File: Ulbra 2023-1/Algortimos e Programação/Aula 16 (Listas)/exerc4.py
#4- Faça um programa que leia uma lista de 5 elementos inteiros e mostre a soma dos elementos lidos.

numeros = []

for i in range(5):
    numero = int(input("Digite um número inteiro: "))
    numeros.append(numero)
    soma = sum(numeros)

print(f"\nA soma dos números inteiros digitados é: {soma}\n")